src/util/RegistrationFormFactory.js

```diff
@@ -142,6 +142,16 @@ define([
       inputOptions.params = {
         'icon': 'person-16-gray'
       };
+      inputOptions.events = {
```

Comment:
probably better to move it to the controller. Because you have one field depends on the other field. The controller should be the better place to coordinate the logic.

Reply:
I thought about that, but i kept it here for consistency since we create the inputs in the formFactory and have listed out the events for the password field here as well. I'll try to move it to the controller and see if its better.